reviews/templates/reviews/decision_form.html

```diff
@@ -24,6 +24,13 @@ <h2>{% trans "Aanvraag beoordelen" %}</h2>
                             De aanvraag is <a href="{{ pdf_url }}" target="_blank">hier</a> in te zien (downloadt als PDF).
                         {% endblocktrans %}
                     </p>
+                    {% if decision.is_final_decision %}
+                        <p>
+                            {% blocktrans trimmed %}
+                                <strong>Attentie:</strong> jij bent de laatste beoordelaar van deze aanvraag. Daarom kun je na het versturen je oordeel en commentaar niet meer bewerken. Controleer je antwoorden goed.
+                            {% endblocktrans %}
```

Comment:
You might want to consider using an `alert` box instead. To draw more attention to it.

Reply:
An alert does look better, good idea